src/Apps/Sell/Routes/__tests__/ThankYouRoute.jest.tsx

```diff
@@ -0,0 +1,15 @@
+import { ThankYouRoute } from "Apps/Sell/Routes/ThankYouRoute"
+import { mount } from "enzyme"
```

Comment:
Let's use `render` or `setupTestWrapperTL` from `DevTools/setupTestWrapper`.

```suggestion
import { render } from "DevTools/renderWithMockBoot"

import { setupTestWrapperTL } from "DevTools/setupTestWrapper"

```